HISTORY.rst

```diff
@@ -25,6 +25,55 @@ unreleased
   are supported. Once registered, the device will show up in the linked IoT Hub and can be interacted with or
   simulated using other `az iot device` commands.
 
+**IoT Central updates**
+
+* Add support for device groups CRUD.
+  - az iot central device-group
+    - az iot central device-group list
+    - az iot central device-group show
+    - az iot central device-group create
+    - az iot central device-group delete
+    - az iot central device-group update
+
+* Add support for device attestation CRUD.
+  - az iot central device attestation
+    - az iot central device attestation show
+    - az iot central device attestation create
+    - az iot central device attestation delete
+    - az iot central device attestation update
+
+* Add support for device/module properties CRUD.
+  - az iot central device list-components
+  - az iot central device show-telemetry-value
+  - az iot central device show-component-telemetry-value
+  - az iot central device properties
+    - az iot central device properties show
+    - az iot central device properties replace
+    - az iot central device properties update
+    - az iot central device properties show-component-properties
+    - az iot central device properties replace-component-properties
+    - az iot central device properties update-component-properties
+  - az iot central device edge module list-components
+  - az iot central device edge module show-telemetry-value
+  - az iot central device edge module show-component-telemetry-value
+  - az iot central device edge module properties
+    - az iot central device edge module properties show
+    - az iot central device edge module properties replace
+    - az iot central device edge module properties update
+    - az iot central device edge module properties show-component-properties
+    - az iot central device edge module properties replace-component-properties
+    - az iot central device edge module properties update-component-properties
+
+* Add support for 2022-05-31 GA version.
```

Comment:
is there any difference btw v1 and 2022-05-31 GA from the user's view? (like how devices are formatted, etc)

Reply:
No. Azure API review board requires fully backward compatibility so every format, payload is the same from users' view. We support some new optional parameters and other new APIs in new GA version though.